### DISCUSSION TITLE
Where is this located?

### DISCUSSION THREAD

**Discussion by Hydraph** [Reloaded-Project/Reloaded-II#413] [Q&A] [answered]
Hello all,
I'm relatively new to modding and wanted to try to mod a few games using reload by following a guide. The guide states to download Reloaded II and to open reloaded-II.exe however I cannot seem to find it anywhere. Can anybody please tell me what's going on or if im missing something super obvious? Thank you!

**Comment by Sewer56** [comment, marked as answer]
Try [Setup.exe](https://github.com/Reloaded-Project/Reloaded-II/releases/download/1.27.11/Setup.exe) from the [Releases](https://github.com/Reloaded-Project/Reloaded-II/releases/tag/1.27.11) page

**Reply by Hydraph** [reply]
Solved it, Thank you!
